pkg/pillar/cmd/volumemgr/handledownloader.go

```diff
@@ -105,6 +108,24 @@ func MaybeRemoveDownloaderConfig(ctx *volumemgrContext, imageSha string) {
 	log.Functionf("MaybeRemoveDownloaderConfig done for %s", imageSha)
 }
 
+func retryDownload(ctx *volumemgrContext, imageSha string) {
+	m := lookupDownloaderConfig(ctx, imageSha)
+	if m == nil {
+		log.Functionf("retryDownload: config missing for %s",
+			imageSha)
+		return
+	}
+	if m.RefCount == 0 {
+		log.Fatalf("retryDownload: Attempting to retry when "+
```

Comment:
Do we know that this can never happen? 
If the verifier is running and detects that the sha is mismatched and at about the same time zedagent starts the process of deleting the content tree then couldn't the refcount drop to zero? A log.Warnf seems like a better choice.